docs/video-miners/core-concepts/winning-ticket-concepts.md

```diff
@@ -0,0 +1,79 @@
+---
+Title: Tickets
+---
+
+The following provides a detailed high level overview about tickets and how their attributes impact orchestrators including but not limited to:
+
+- What are tickets and how do they work
+- How adjusting orchestrator attributes affecting tickets impact orchestator payouts
+- How orchestrator behavior changes with incoming tickets
+
+***Related documentation*** [Earnings](https://docs.livepeer.org/video-miners/core-concepts/earnings) and [Payments](https://docs.livepeer.org/video-miners/core-concepts/payments).
+
+## What is a ticket?
+
+Broadcasters using the Livepeer network attach a “ticket” to each segment of a video they want transcoded. A ticket can be thought of as a form of work completed for a given video stream a broadcaster is emitting to the Livepeer network.  The final number of tickets associated with a given stream is a product of the fees proposed by the orchestrator for doing the work and the total length of the video processed.
+
+A ticket is comprised of the following attributes:
+
+- **recipient**: ETH address of recipient (orchestrator completing transcoding work)
+- **faceValue**: The ETH value of the ticket payable to the recipient (orchestrator) if the ticket wins
```

Comment:
how is this value derived? its set by the broadcaster but how is it set for each ticket?
how is this related to the `maxFaceValue` parameter set by orchestrators?